src/basic/Matrix.js

```diff
@@ -672,12 +672,11 @@ var Matrix = Base.extend(/** @lends Matrix# */{
     },
 
     /**
-     * Attempts to decompose the affine transformation described by this matrix
-     * into `scaling`, `rotation` and `skewing`, and returns an object with
-     * these properties if it succeeded, `null` otherwise.
+     * Decompose the affine transformation described by this matrix into
```

Comment:
@sapics minor spelling mistake: `Decomposes`. After that it's good to go! Thanks for spotting this!

Reply:
Thanks for noticing my mistake in grammar!